server/athenian/api/controllers/miners/github/contributors.py

```diff
@@ -12,21 +13,33 @@
     PullRequestReview, PushCommit, Release, User
 
 
+async def mine_contributors(repos: Collection[str],
+                            db: databases.Database,
+                            time_from: Optional[datetime] = None,
+                            time_to: Optional[datetime] = None,
+                            with_stats: Optional[bool] = True,
+                            cache: Optional[aiomcache.Client] = None) -> List[dict]:
+    """Discover developers who made any important action in the given repositories and \
+    in the given time frame."""
+    time_from = time_from or datetime(1970, 1, 1, tzinfo=timezone.utc)
+    time_to = time_to or datetime.now(timezone.utc)
+
+    return await _mine_contributors(repos, time_from, time_to, db, with_stats, cache)
+
+
 @cached(
     exptime=5 * 60,
     serialize=marshal.dumps,
     deserialize=marshal.loads,
     key=lambda repos, time_from, time_to, **_: (
         ",".join(repos), time_from.timestamp(), time_to.timestamp()),
 )
-async def mine_contributors(repos: Collection[str],
-                            time_from: datetime,
-                            time_to: datetime,
-                            db: databases.Database,
-                            cache: Optional[aiomcache.Client],
-                            ) -> Tuple[dict, dict]:
-    """Discover developers who made any important action in the given repositories and \
-    in the given time frame."""
+async def _mine_contributors(repos: Collection[str],
+                             time_from: datetime,
+                             time_to: datetime,
+                             db: databases.Database,
+                             with_stats: bool,
```

Comment:
```suggestion
                             with_stats: bool,
                             db: databases.Database,
```
(since you have to touch the code above - no special meaning here, just a convention)